packages/react-components/src/organisms/SharedResearchCard.tsx

```diff
@@ -109,6 +119,7 @@ const SharedResearchCard: React.FC<SharedResearchCardProps> = ({
         />
       )}
     </div>
+    {showTags && keywords.length > 0 && <TagList max={3} tags={keywords} />}
```

Comment:
I believe it's necessary to add a marginTop of 24
<img width="623" alt="Screenshot 2024-03-25 at 14 02 22" src="https://github.com/yldio/asap-hub/assets/16595804/50003ee9-7a34-4dcd-8497-f8c85217c295">